resources/ext/ezbake.conf

```diff
@@ -21,8 +21,9 @@ ezbake: {
                  "/opt/puppetlabs/puppet/bin/puppet config set --section master rundir  /var/run/puppetlabs/puppetserver",
                  "/opt/puppetlabs/puppet/bin/puppet config set --section master pidfile /var/run/puppetlabs/puppetserver/puppetserver.pid",
                  "/opt/puppetlabs/puppet/bin/puppet config set --section master codedir /etc/puppetlabs/code",
-                 "usermod --home /opt/puppetlabs/server/data/puppetserver puppet",
-                 "install --directory --owner=puppet --group=puppet --mode=775 /opt/puppetlabs/server/data"
+                 "install --directory /etc/puppetlabs/puppet/ssl",
+                 "chown -R puppet:puppet /etc/puppetlabs/puppet/ssl",
+                 "find /etc/puppetlabs/puppet/ssl -type d -exec chmod 770 {} \;",
```

Comment:
I'm getting the following error trying to run `lein with-profile ezbake ezbake stage`, which exercises these changes to the configuration file:

```
Error encountered performing task 'ezbake' with profile(s): 'ezbake'
com.typesafe.config.ConfigException$Parse: Reader: 26: in value for key 'postinst': backslash followed by ';', this is not a valid escape sequence (quoted strings use JSON escaping, so use double-backslash \\ for literal backslash)
```

My suggestion is to try this, which should be a bit more robust with shell escape issues:

```bash
find  /etc/puppetlabs/puppet/ssl -type d -print0 | xargs -0 chmod 770
```

Also, I think the trailing comma is illegal since this is JSON.